app/src/main/java/com/hedvig/app/feature/addressautocompletion/ui/AddressAutoCompleteScreen.kt

```diff
@@ -129,10 +134,31 @@ private fun AddressInput(
             horizontalAlignment = Alignment.CenterHorizontally,
             modifier = Modifier.padding(horizontal = 16.dp, vertical = 8.dp),
         ) {
+            var textFieldValue by remember {
+                val text = viewState.input.rawText
+                mutableStateOf(
+                    TextFieldValue(
+                        text = text,
+                        selection = TextRange(text.length)
+                    )
+                )
+            }
```

Comment:
Holding the data internally. Important detail here is that the `remember` has no key, therefore the TextFieldValue gets populated once with the initial data that comes from the viewModel.